### ISSUE TITLE
Request: Flatpak release

### ISSUE THREAD

**Issue by chrisdeburg** [mmatyas/openblok#65]
I loooove your game and would be soo thrilled to see it spread to the fabulous Batocera retro distribution
https://flatpak.org/

**Comment by chrisdeburg** [comment]
Tried creating a flatpak for Batocera running on the Raspberyy Pi myself inspired by SteinZone*, but ran into trouble with the SDL2 library :-/
* https://www.steinzone.de/wordpress/how-to-build-a-flatpak-of-a-sdl2-game-commander-genius/

"Failed to clone 'thirdparty/libSDL2pp' a second time, aborting"

**Comment by chrisdeburg** [comment]
https://docs.flathub.org/docs/for-app-authors/submission/

**Comment by mmatyas** [comment]
Ok, this Flatpak file should be buildable! https://github.com/mmatyas/openblok-flatpak

**Comment by chrisdeburg** [comment]
Thanks Mátyás!

I downloaded the yml file and SharedModules (tried first without, but my machine complained about missing files).

I've got this far on Ubuntu 22 and will try tomorrow on a Raspberry Pi

[openblok-debug.txt](https://github.com/mmatyas/openblok/files/15195675/openblok-debug.txt)

**Comment by chrisdeburg** [comment]
Yes! For Batocera v42 I took your AppImage v0.8.7 and copied the binary file (remember to make it executable by chmod +x) and shared folder to
/userdata/roms/ports/.data/openblok

And created a file, /userdata/roms/ports/OpenBlok.sh, with the following:
DIR="$(dirname "$(readlink -f "$0")")"
cd "${DIR}/.data/openblok"
./openblok

Thanks soo much for this wonderful game!

PS Tried to compile on Debian 13 (ARM) to have your game on my Raspberry Pi, but ran into problems and hope there some day will be an AppImage for ARM64
